database/migrations/2024_07_01_115148_create_internship_attributes_table.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration {
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create('internship_attributes', function (Blueprint $table) {
+            $table->id();
+
+            $table->string('key')->comment('The possible values are: year, language, track');
```

Comment:
This might also be done with an enum: `$table->enum('key', ['year', 'language', 'track'])`. Adv.: data integrity, Disadv.: a little less extendability.